src/components/pinning-manager/PinningManager.js

```diff
@@ -0,0 +1,151 @@
+
+  const [sortSettings, setSortSettings] = useState({
+    sortBy: 'addedAt',
+    sortDirection: SortDirection.ASC
+  })
+  useEffect(() => {
+    (async () => {
+      await doFilesFetch()
+      await doFilesSizeGet()
+    })()
+  }, [doFilesFetch, doFilesSizeGet])
+
+  const localPinning = useMemo(() =>
+    ({ name: t('localPinning'), type: 'LOCAL', svgIcon: GlyphPin, totalSize: filesSize }),
+  [filesSize, t])
+
+  const sortedServices = useMemo(() =>
+    (pinningServices || []).sort(sortByProperty(sortSettings.sortBy, sortSettings.sortDirection === SortDirection.ASC ? 1 : -1)),
+  [pinningServices, sortSettings.sortBy, sortSettings.sortDirection])
+
+  const sortedList = useMemo(() => [localPinning, ...sortedServices], [localPinning, sortedServices])
+
+  return (
+    <Fragment>
+      <div className="mv4 pinningManager">
+        <div className='ph4 flex items-center bg-white lh-copy charcoal f6 fw5'>
+          <AutoSizer disableHeight>
+            {({ width }) => (
+              <Table
+                className='tl fw4 w-100 f6'
+                headerClassName='gray ttc fw4 f6 ph2'
+                width={width}
+                height={(sortedList.length + 1) * ROW_HEIGHT}
+                headerHeight={HEADER_HEIGHT}
+                rowHeight={ROW_HEIGHT}
+                rowCount={sortedList.length}
+                rowGetter={({ index }) => sortedList[index]}
+                rowClassName='bb b--light-gray mt2'
+                sort={(...sortArgs) => setSortSettings(...sortArgs)}
+                sortBy={sortSettings.sortBy}
+                sortDirection={sortSettings.sortDirection}>
+                <Column label={t('service')} dataKey='name' width={250} cellRenderer={ServiceCell} className='charcoal truncate f6 pl2' />
+                <Column label={t('files')} dataKey='totalSize' width={250} cellRenderer={SizeCell} className='charcoal truncate f6 pl2' />
+                <Column label={t('bandwidthUsed')} dataKey='bandwidth' width={250} cellRenderer={BandwidthCell} className='charcoal truncate f6 pl2' />
+                <Column label={t('autoUpload')} dataKey='autoUpload' width={250} cellRenderer={({ rowData }) => <AutoUploadCell autoUpload={rowData.autoUpload} type={rowData.type} t={t} />} className='pinningManagerColumn charcoal truncate f6 pl2' />
+              </Table>
+            )}
+          </AutoSizer>
+        </div>
+        {/* <div className='flex justify-end w-100 mt2'>
+          <Button className="tc mt2" bg='bg-navy' onClick={onModalOpen}>
+            <span><span className="aqua">+</span> {t('actions.addService')}</span>
+          </Button>
+        </div> */}
+      </div>
+
+      <Overlay show={isModalOpen} onLeave={onModalClose}>
+        <PinningModal className='outline-0' onLeave={onModalClose} t={t} />
+      </Overlay>
+    </Fragment>
+  )
+}
+
+PinningManager.defaultProps = {
+  pinningServices: []
+}
+
+const ServiceCell = ({ rowData }) => {
+  const iconClass = 'mr2 pr1 fill-teal'
```

Comment:
```suggestion
  const iconClass = 'mr2 pr1 fill-aqua'
```
Color consistency was my fault, sorry :( 

Also, can the icon please not scale as the page width becomes narrower?

Reply:
Done!